static/app/views/settings/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -132,6 +139,31 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
     return this.metadata.features;
   }
 
+  renderTabs() {
+    // TODO: Convert to styled component
+    const {integrationSlug} = this.props.params;
+    const {organization} = this.props;
+    // TODO(cathy): remove feature check
+    const tabs =
+      integrationSlug === 'github' && organization.features.includes('pr-comment-bot')
+        ? this.tabs
+        : this.tabs.filter(tab => tab !== 'features');
+
+    return (
+      <ul className="nav nav-tabs border-bottom" style={{paddingTop: '30px'}}>
+        {tabs.map(tabName => (
+          <li
+            key={tabName}
+            className={this.state.tab === tabName ? 'active' : ''}
+            onClick={() => this.onTabChange(tabName)}
+          >
+            <CapitalizedLink>{this.getTabDisplay(tabName)}</CapitalizedLink>
```

Comment:
huh. we're capitalizing via css? 🤷 
lol we could also do this via JS right?

Reply:
i'm not sure lol i copy pasted from the parent component

Reply:
it's a capitalized link

Reply:
this capitalizes it visually but keeps the text in the source code as lowercase